Client/Configuration/FirefoxNightly.xcconfig

```diff
@@ -2,7 +2,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-MOZ_BUNDLE_DISPLAY_NAME = Firefox 🌙
+MOZ_BUNDLE_DISPLAY_NAME = Firefox
```

Comment:
I wonder if we should just call this Nightly instead of Firefox. The icon might be enough for users to know that it's Firefox already.